package/base-files/files/etc/init.d/done

```diff
@@ -11,6 +11,24 @@ boot() {
 		sh /etc/rc.local
 	}
 
+	[ -d /sys/firmware/efi ] && {
```

Comment:
Why is the directory `/sys/firmware/efi` a prerequisite for this?  Shouldn't the prerequisite be the use of GPT, which is entirely possible with a traditional PC-BIOS system?  Systems without EFI were entirely capable of using GPTs.